### ISSUE TITLE
[Regression]: "Playwright Test for VSCode" v1.1.1 extension adds config to `.vscode\settings.json` file

### ISSUE THREAD

**Issue by kosteklvp** [microsoft/playwright#30780]
### Last Good Version

1.0.22

### First Bad Version

1.1.1

### Steps to reproduce

1. Open "TESTING: PLAYWRIGHT" view.
2. Select project in "PROJECTS" section.
![obraz](https://github.com/microsoft/playwright/assets/29569674/903758e2-32dd-4afe-8026-c9e0532537e1)

### Expected behavior

Nothing was added to `.vscode/settings.json` file, when changing default project.

### Actual behavior

This section is added to `.vscode/settings.json` file.
![obraz](https://github.com/microsoft/playwright/assets/29569674/55b0007c-7742-43c5-93cc-b685b5507926)

I don't think this is a good place to save this information, because this file can be tracked by VCS.

### Environment

```shell
System:
    OS: Windows 11 10.0.22631
  Binaries:
    Node: 20.12.2 - C:\Program Files\nodejs\node.EXE
    npm: 10.5.0 - C:\Program Files\nodejs\npm.CMD
  IDEs:
    VSCode: 1.88.1
  Languages:
    Bash: 5.1.16
  npmPackages:
    @playwright/test: ^1.40.0 => 1.43.1
```

**Comment by mxschmitt** [comment]
Sounds like we should switch to workspaceState instead: https://code.visualstudio.com/api/extension-capabilities/common-capabilities#data-storage

**Comment by mxschmitt** [comment]
@kosteklvp this should be fixed in v1.1.2
